cooperator/models/partner.py

```diff
@@ -139,7 +141,7 @@ def _compute_share_info(self):
     )
     old_member = fields.Boolean(
         string="Old cooperator",
-        help="Check this box if this cooperator is" " no more an effective member.",
+        help="Check this box if this cooperator is no more an effective member.",
```

Comment:
would be better english, indeed. is there a document where all renaming suggestions are listed together? that  would be the place for this, as it would be part of a separate change.

Reply:
Don't know about such a list. Maybe @robinkeunen knows where to keep track of those ?

Reply:
issue opened here https://github.com/coopiteasy/vertical-cooperative/issues/386